CCCS_YARA.yml

```diff
@@ -182,15 +184,19 @@ category:
       child: info|exploit|technique|tool|malware
 
 info|exploit|technique|tool|malware:
-   description: 'Specific information about the category'
+   description: 'Specific information about the category.
+                  if_key_gen_mitre_att: is used to define which keys will been used to generate mitre_att software ids
+                  mitre_softwareid_gen: is an array that will be used during runtime to store software ids generated,
+                                        this will be compared with mitre_softwareid_found and values not found there
+                                        will be generated'
    format: 'uppercase'
    unique: No
    optional: Yes
    validator: valid_category_type
    argument:
       parent: category
-      ifKeyGenMitreAtt: malware
-      malwareIdsFound:
+      if_key_gen_mitre_att: ^tool$|^malware$
+      mitre_softwareid_gen: []
```

Comment:
should not be in the config file since it will never be populated unless at runtime